static/src/javascripts/projects/common/modules/identity/consent-journey.js

```diff
@@ -16,10 +16,10 @@ const showJourney = (journeyEl: HTMLElement): Promise<void> =>
 const hideLoading = (loadingEl: HTMLElement): Promise<void> =>
     fastdom.write(() => loadingEl.remove());
 
-const shouldDisplaySectionWarning = (
+const uncheckedBoxes = (
```

Comment:
This name is a little misleading as it isn't just the unchecked boxes

Reply:
It is now

Reply:
i'm not digging this new structure, i'd rather keep the old structure and remove the wrapper methods so you can do 

```
Promise.all([
  shouldDisplaySectionWarning(journeyEl, 'marketing-consents'),
  shouldDisplaySectionWarning(journeyEl, 'email');
]).then(checkboxes => [].concat(...checkboxes)) 
```

and then merge those, instead of having a method taking an array. You could even get fancy and do